includes/services/GoogleMaps3/Maps_GoogleMaps3.php

```diff
@@ -222,60 +223,63 @@ public function addParameterInfo( array &$params ) {
 			'message' => 'maps-googlemaps3-par-enable-fullscreen',
 		);
 	}
-	
+
 	/**
 	 * @see iMappingService::getDefaultZoom
-	 * 
+	 *
 	 * @since 0.6.5
-	 */	
+	 */
 	public function getDefaultZoom() {
 		global $egMapsGMaps3Zoom;
 		return $egMapsGMaps3Zoom;
-	}	
-	
+	}
+
 	/**
 	 * @see MapsMappingService::getMapId
-	 * 
+	 *
 	 * @since 0.6.5
 	 */
 	public function getMapId( $increment = true ) {
 		static $mapsOnThisPage = 0;
-		
+
 		if ( $increment ) {
 			$mapsOnThisPage++;
 		}
-		
+
 		return 'map_google3_' . $mapsOnThisPage;
-	}	
-	
+	}
+
 	/**
 	 * Returns the names of all supported map types.
-	 * 
+	 *
 	 * @return array
 	 */
 	public static function getTypeNames() {
 		return array_keys( self::$mapTypes );
 	}
-	
+
 	/**
 	 * Returns the names of all supported map layers.
-	 * 
+	 *
 	 * @since 1.0
-	 * 
+	 *
 	 * @return array
 	 */
 	public static function getLayerNames() {
 		return self::$mapLayers;
-	}	
-	
+	}
+
 	/**
 	 * @see MapsMappingService::getDependencies
-	 * 
+	 *
 	 * @return array
 	 */
 	protected function getDependencies() {
 		return array(
-			self::getApiScript( $GLOBALS['wgLang']->getCode() ),
+			self::getApiScript(
+				 $GLOBALS['egMapsGMaps3Language'] instanceof \Language ?
+				 $GLOBALS['egMapsGMaps3Language']->getCode() : $GLOBALS['egMapsGMaps3Language']
```

Comment:
I do remember that `wgLang`/`wgContLang` have some weird `StubUserLang` object when initialized therefore changing the check to:

```
		return array(
			self::getApiScript(
				is_string( $GLOBALS['egMapsGMaps3Language'] ) ?
				$GLOBALS['egMapsGMaps3Language'] : $GLOBALS['egMapsGMaps3Language']->getCode()
			)
```
which will also pass the tests.